org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/dom/DOMElement.java

```diff
@@ -287,11 +287,15 @@ public boolean isInStartTag(int offset) {
 	}
 
 	public boolean isInEndTag(int offset) {
+		return isInEndTag(offset, false);
+	}
+
+	public boolean isInEndTag(int offset, boolean afterBackSlash) {
 		if (endTagOpenOffset == NULL_VALUE) {
 			// case >|
 			return false;
 		}
-		if (offset > endTagOpenOffset && offset < getEnd()) {
+		if (offset > endTagOpenOffset + (afterBackSlash ? 1 : 0) && offset < getEnd()) {
 			// case <\bean | >

```

Comment:
`</bean | >`

Reply:
fixed

Reply:
Hey Angelo, I think you got the wrong line, I was talking about the comment on line 299

Reply:
my bad,sorry-(